bin/kalastatic

```diff
@@ -66,9 +68,10 @@ program
 
       // Set up the server.
       bs.init({
-        server: finalDest,
-        index: 'index.html',
-        serveStatic: ['.']
+        server: base,
```

Comment:
It is handy, the more configs the more easier we can adapt to change. I like the options you added :+1: